Example/AsyncReactorExample/Features/Repository/Search/RepositorySearchReactor.swift

```diff
@@ -12,9 +12,20 @@ import Logging
 
 private let logger = Logger(label: "RepositorySearchReactor")
 
-enum SortOptions: String, CaseIterable {
-    case watchers = "Watchers Count"
-    case forks = "Forks"
+enum SortOptions: String, CaseIterable, Identifiable {
+    var id: Self { return self }
+    
+    case watchers = "watchers"
+    case forks = "forks"
```

Comment:
no need to provide an explicit raw value in this case. default it will be the case name